topics/journalpost.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: journalpost
+  namespace: bidrag
+  labels:
+    team: bidrag
+spec:
+  pool: {{kafka-pool}}
+  config:
+    cleanupPolicy: compact
+    minimumInSyncReplicas: 1
+    partitions: 1
+    replication: 3
+    retentionBytes: -1
```

Comment:
Betyr dette at det ikke er noen grense for hvor mye som lagres?

Reply:
ja